packages/edit-site/src/components/dataviews/view-actions.js

```diff
@@ -39,6 +39,8 @@ const availableViews = [
 		id: 'grid',
 		label: __( 'Grid' ),
 	},
+	// TODO: This(the whole layout) needs be redesigned because in order for it to
+	// work properly, needs extra code from the consumer of DataViews component.
```

Comment:
What is this comment about?

Reply:
Right now, the view actions menu shows the available layouts from here, including the `side by side`. But since this layout needs the consumer to add code, it makes no sense to show it by default, as it does nothing without the extra code.

Makes sense?

Reply:
I don't know, for me it's fine, its design is not ready yet (it needs to be a list) which IMO will remove the confusion.